tests/test_ebuild.py

```diff
@@ -26,7 +26,31 @@ def test_simple(self):
         ebuild.add_run_depend('p2os_driver')
         ebuild.src_uri = 'https://www.website.com/download/stuff.tar.gz'
         ebuild.distro = 'lunar'
-        got_text = ebuild.get_ebuild_text('tests', 'BSD')
+        got_text = ebuild.get_ebuild_text('Open Source Robotics Foundation', 'BSD')
         with open('tests/ebuild/simple_expected.ebuild', 'r') as expect_file:
             correct_text = expect_file.read()
         self.assertEqual(got_text, correct_text)
+
+    def test_bad_build_depend(self):
+        """Test Bad Build Dependency"""
+        ebuild = Ebuild()
+        ebuild.homepage = 'https://www.website.com'
+        ebuild.description = 'an ebuild'
+        ebuild.src_uri = 'https://www.website.com/download/stuff.tar.gz'
+        ebuild.distro = 'lunar'
+        ebuild.add_run_depend('p2os_driver')
+        ebuild.add_build_depend('fake_package', False)
+        with self.assertRaises(UnresolvedDependency):
+            ebuild_text = ebuild.get_ebuild_text('Open Source Robotics Foundation', 'BSD')
+
+    def test_bad_run_depend(self):
+        """Test Bad Build Dependency"""
```

Comment:
s/build/run/

Reply:
Ah! You've caught me with my copypasta! Fixed.